src/commands.rs

```diff
@@ -1137,12 +1137,12 @@ pub fn command_json_clear<M: Manager>(
 
     let paths = find_paths(path, root, |_v| true)?;
```

Comment:
Do we want to align with other commands that handle only matching keys?
```suggestion
    let paths = find_paths(path, root, |v| {
        v.get_type() == SelectValueType::Array || v.get_type() == SelectValueType::Object
    })?;
```